src/osx/carbon/cursor.cpp

```diff
@@ -278,7 +278,28 @@ wxCursor::wxCursor(const wxString& cursor_file, wxBitmapType flags, int hotSpotX
     if ( flags == wxBITMAP_TYPE_MACCURSOR_RESOURCE )
     {
 #if wxOSX_USE_COCOA
-        wxFAIL_MSG( wxT("Not implemented") );
+        wxString fileName;
+        wxImage image;
+        fileName = wxStandardPaths::Get().GetResourcesDir() + "/" + cursor_file + ".png";
+        image.LoadFile( fileName, wxBITMAP_TYPE_PNG );
```

Comment:
This would be more readable
```suggestion
        const wxFileName fileName(wxStandardPaths::Get().GetResourcesDir(), cursor_file, ".png");
        wxImage image(fileName.GetFullPath(), wxBITMAP_TYPE_PNG);
```